cypress/e2e/tests/migration/dynamic-report/issues/singleApplication/sort.test.ts

```diff
@@ -0,0 +1,62 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(["@tier2"], "Issues sort validations", function () {
+    let application: Analysis;
+
+    before("Load data, create Analysis instance and run analysis", function () {
+        login();
+        cy.fixture("application").then(function (appData) {
+            this.appData = appData;
+        });
+        cy.fixture("analysis").then(function (analysisData) {
+            this.analysisData = analysisData;
+        });
+
+        cy.intercept("GET", "/hub/application*").as("getApplication");
+    });
+
+    it("BUG MTA-2067 - Source Analysis on daytrader app and its issues sorting validation", function () {
+        // Create Analysis instance in before hook
```

Comment:
This is not the "before hook", this comment can be removed

Reply:
Fixed